### ISSUE TITLE
Alpine.js conflict

### ISSUE THREAD

**Issue by twitcher07** [nystudio107/craft-seomatic#911]
### Describe the bug
I ran into a bug with a site I was working on that uses Alpine.js and SEOmatic with the gtag.js integration. The "Body Script Render Location" was set to "Body begin". 

The body tag for the site was this:
```html
<body class="{% if bodyClass is defined %}{{ bodyClass }}{% endif %}"
        x-data="{ isOpen: false }"
        x-on:keydown.escape="isOpen = false"
        x-on:resize.window.debounce="isOpen = window.outerWidth > 1024 ? false : isOpen;"
        x-bind:class="{ 'overflow-hidden nav-is-open' : isOpen }">
 ```

But it was rendering as broken HTML:
```html
<body class="" 
    x-data="{ isOpen: false }" 
    x-on:keydown.escape="isOpen = false" 
    x-on:resize.window.debounce="isOpen = window.outerWidth ><script async src=" https:="" www.googletagmanager.com="" gtag="" js?id="UA-69945126-1&quot;">
```
I assume it's because of my window resize event has a ternary operator with a ">" and SEOmatic assumes that's the end of the body tag. Because that's exactly where it's injecting the script tag.

Work around for me right now is to set the body render location to "body end".

### To reproduce
1. Add alpine attribute that contains ">".
2. Set gtag.js to render at "body begin".

### Expected behaviour
Script tag should prepend after the opening of the body tag.

### Versions
- Plugin version: 3.3.42
- Craft version: 3.6.14

**Comment by khalwat** [comment]
So this isn't an SEOmatic thing -- this is a Yii2 & Craft CMS thing. Check out:

https://github.com/craftcms/cms/issues/7779

...for details. P&T changed the pattern matching in Craft 3.7 such that what you're doing should no longer cause an issue. But it won't be fixed until Craft 3.7.

**Comment by twitcher07** [comment]
@khalwat Oh ok. Thanks for letting me know.
